### ISSUE TITLE
Tag and release for Rosetta support

### ISSUE THREAD

**Issue by dgjustice** [networktocode/yangify#16]
@jathanism It doesn't look like much has happened in the last few months, and I am wondering if there is anything that needs to be done before tagging a release.  Per @dbarrosop's [comment](https://github.com/networktocode/ntc-rosetta/pull/26#issuecomment-528736775) on ntc-rosetta, I need the new fields on the Yangify develop branch to refactor that PR the way he wants to see it.  It would be a large deviation from what I originally did, so I'll simply create a new PR if we can upgrade to a new Yangify version.  Let me know if there is anything I can do to assist.

**Comment by FragmentedPacket** [comment]
Yangify 0.1.2 has been released and should encompass the stuff you need.
